ui-v2/src/components/ui/tag-badge-group.tsx

```diff
@@ -3,14 +3,14 @@ import { HoverCard, HoverCardContent, HoverCardTrigger } from "./hover-card";
 import { TagBadge } from "./tag-badge";
 
 type TagBadgeGroupProps = {
-	tags: string[];
+	tags: string[] | undefined;
```

Comment:
Added for + dev x. 
Schema says tags is optional, so handling `undefined` to default to an empty string